caproto/curio/server.py

```diff
@@ -32,7 +32,6 @@ class AsyncLibraryLayer:
 
 class CurioAsyncLayer(AsyncLibraryLayer):
     name = 'curio'
-    sleep = curio.sleep
```

Comment:
no naps?

Reply:
ah, I see, expect the library to provide `sleep`

Reply:
Fix here could be `sleep = staticmethod(curio.sleep)` if we think it's desirable to increase the scope of this 'async library layer'

Reply:
no, I think we should only shim out when we have to and I strongly suspect everyone has 'sleep'.